### PR TITLE
Import script

### PR DESCRIPTION
**Pull request by juniormp**
## Import Transfer Script

This code reads data from a CSV file, connects to a database, and processes the data to insert missing transfers into the database. 

It first retrieves existing transaction hashes from the database, then filters the CSV data to find missing transfers. 

These missing transfers are formatted into the expected database structure and inserted into the database in batches, ensuring efficient data insertion while handling any errors during the process.

### Mapping from csv to collections

```
  const formattedMissingTransfers = missingTransfers.map((transfer) => {
    return {
      TxId: transfer.hash.substring(1, 8),
      chainId: "eip155:137",
      tokenSymbol: "g1",
      tokenAddress: "0xe36BD65609c08Cd17b53520293523CF4560533d0",
      senderTgId: "",
      senderWallet: transfer.from,
      senderName: "",
      recipientTgId: "",
      recipientWallet: transfer.to,
      tokenAmount: transfer.value,
      transactionHash: transfer.hash,
      dateAdded: new Date(transfer.block_time),
    };
  });
```

## Import Backup Script

This script initiates a backup process for transferring documents from one MongoDB collection `transfers` to another `transfers-backup.` 

Fetches all documents from the source collection, and then inserts them into the destination collection. 
The script ensures that the backup process only proceeds if there are documents to transfer, and it exits gracefully after completing the backup operation, logging a message to indicate its completion.

### CODE REVIEW

File: src/scripts/import-transfer.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  );
+  console.log(
+    "Number of missing transfers in `transfers` collection: ",
+    missingTransfers.length
+  );
+
+  // Step 3: Format missing transfers
+  const formattedMissingTransfers = missingTransfers.map((transfer) => {
+    return {
+      txId: "",
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:33]: Here two things:
- `txId` must be replaced by `TxId` to match the other elements in the database.
- can you complete this field, these are the first letters of the transaction hash, without the 0 in front, i.e. `transactionHash.substring(1, 8)`

File: src/scripts/import-transfer.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      chainID: "eip155:137",
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:34]: `chainID` should be `chainId`

File: src/scripts/import-transfer.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      tokenSymbol: "g1",
+      tokenAddress: "0xe36BD65609c08Cd17b53520293523CF4560533d0",
+      senderTelegramID: "",
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:37]: `senderTelegramID` should be `senderTgId`

File: src/scripts/import-transfer.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+      senderWalletAddress: transfer.from,
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:38]: `senderWalletAddress` should be `senderWallet`

File: src/scripts/import-transfer.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+      senderName: "",
+      recipientTelegramID: "",
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:40]: `recipientTelegramID` should be `recipientTgId`

File: src/scripts/import-transfer.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+      recipientWalletAddress: transfer.to,
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:41]: `recipientWalletAddress` should be `recipientWallet`

File: src/scripts/import-transfer.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+      senderMessage: "",
+      tokenAmount: transfer.value,
+      timestamp: new Date(transfer.block_time),
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:44]: Let us replace `timestamp` by `dateAdded` in order to be consistent with the other elements of the database and be able to do proper comparisons.

File: src/scripts/import-transfer.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+      transactionHash: transfer.hash,
+      isActive: true,
```
Comment (tcoratger) [inline src/scripts/import-transfer.js:46]: What does it correspond to?

Comment (juniormp) [reply src/scripts/import-transfer.js:46]: deleted

**Pull request closed without merging**